.github/workflows/ghcr-cleanup.yml

```
@@ -33,7 +33,11 @@ permissions:
 
 env:
   SERVICE_IMAGES: "insight-analytics insight-authenticator insight-frontend insight-gateway insight-identity-resolution insight-toolbox insight-ui-tests"
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)